indigo_app/views/users.py

```diff
@@ -65,3 +66,16 @@ def get_object(self, queryset=None):
 
     def get_success_url(self):
         return get_request_param(self.request, self.get_redirect_field_name(), settings.LOGIN_REDIRECT_URL)
+
+
+class UserPopupView(AbstractAuthedIndigoView, DetailView):
+    model = User
+    context_object_name = 'user'
+    slug_field = 'username'
+    slug_url_kwarg = 'username'
+    template_name = 'indigo_app/user_popup.html'
+    queryset = User.objects.all()
```

Comment:
```suggestion
    queryset = User.objects
```

`.all()` goes to the database, which means it'll load all users in the DB when the app starts up, and then never refresh.